AppCenter/AppCenter/Internals/Storage/MSDBStorage.m

```diff
@@ -112,6 +113,17 @@ + (void)setVersion:(NSUInteger)version inOpenedDatabase:(void *)db {
   [MSDBStorage executeNonSelectionQuery:query inOpenedDatabase:db];
 }
 
++ (void)enableAutoVacuumInOpenedDatabase:(void *)db {
+  NSArray<NSArray *> *result = [MSDBStorage executeSelectionQuery:@"PRAGMA auto_vacuum" inOpenedDatabase:db];
+  int vacuumMode = [(NSNumber *)result[0][0] intValue];
+  BOOL autoVacuumDisabled = vacuumMode == 0;
+  [MSDBStorage executeNonSelectionQuery:@"PRAGMA auto_vacuum = FULL;" inOpenedDatabase:db];
+  if (autoVacuumDisabled) {
+    MSLogDebug([MSAppCenter logTag], @"Vacuuming database to enable auto_vacuum");
```

Comment:
I suggest write a comment before line 122 to explain why `auto_vacuum` works this way.

There are two scenarios covered: (1) if auto vacuum is enabled, use `PRAGMA auto_vacuum = FULL` ; (2) if auto vacuum is disabled, first use `PRAGMA auto_vacuum = FULL` to set the state, then invoke the VACUUM command to reorganize the entire database file. 

See [auto_vacuum documentation](https://www.sqlite.org/pragma.html#pragma_auto_vacuum) 
```The database connection can be changed between full and incremental autovacuum mode at any time. However, changing from "none" to "full" or "incremental" can only occur when the database is new (no tables have yet been created) or by running the VACUUM command. To change auto-vacuum modes, first use the auto_vacuum pragma to set the new desired mode, then invoke the VACUUM command to reorganize the entire database file.```